src/udev/udev-manager.c

```diff
@@ -1221,6 +1223,10 @@ int manager_init(Manager *manager, int fd_ctrl, int fd_uevent) {
         if (r < 0)
                 return log_error_errno(r, "Failed to bind udev control socket: %m");
 
+        r = manager_open_varlink(manager);
```

Comment:
In this stage, Manager.event is not set, but `manager_open_varlink()` calls `varlink_server_attach_event()`.
Maybe, better to set sd-event earlier in `manager_new()`.